api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -217,33 +214,60 @@ void shouldSendNewChildEmailWhenPaymentCycleIncludesBackdatedVouchers() throws J
         // setup some claim variables
         String cardAccountId = UUID.randomUUID().toString();
         List<LocalDate> sixWeekOldAndThreeYearOld = asList(LocalDate.now().minusWeeks(6), THREE_YEAR_OLD);
-        int cardBalanceInPenceBeforeDeposit = 88;
 
         wiremockManager.stubSuccessfulEligibilityResponse(sixWeekOldAndThreeYearOld);
-        wiremockManager.stubSuccessfulCardBalanceResponse(cardAccountId, cardBalanceInPenceBeforeDeposit);
+        wiremockManager.stubSuccessfulCardBalanceResponse(cardAccountId, CARD_BALANCE_IN_PENCE_BEFORE_DEPOSIT);
         wiremockManager.stubSuccessfulDepositResponse(cardAccountId);
         stubNotificationEmailResponse();
 
-        Claim claim = createClaimWithPaymentCycleEndingYesterday(cardAccountId, asList(THREE_YEAR_OLD), LocalDate.now().minusWeeks(7));
+        Claim claim = createClaimWithPaymentCycleEndingYesterday(cardAccountId, singletonList(THREE_YEAR_OLD), LocalDate.now().minusWeeks(7));
 
         invokeAllSchedulers();
 
         // confirm new payment cycle created with a payment
         PaymentCycle newCycle = repositoryMediator.getCurrentPaymentCycleForClaim(claim);
         PaymentCycleVoucherEntitlement expectedVoucherEntitlement =
                 aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(LocalDate.now(), sixWeekOldAndThreeYearOld);
-        assertPaymentCycleIsFullyPaid(newCycle, sixWeekOldAndThreeYearOld, cardBalanceInPenceBeforeDeposit, expectedVoucherEntitlement);
+        assertPaymentCycleIsFullyPaid(newCycle, sixWeekOldAndThreeYearOld, expectedVoucherEntitlement);
 
         // confirm card service called to make payment
         Payment payment = newCycle.getPayments().iterator().next();
-        wiremockManager.assertThatGetBalanceRequestMadeForClaim(payment);
+        wiremockManager.assertThatGetBalanceRequestMadeForClaim(payment.getCardAccountId());
         wiremockManager.assertThatDepositFundsRequestMadeForClaim(payment);
 
         // confirm notify component invoked with correct email template & personalisation
         assertThatNewChildEmailWasSent(newCycle);
         verifyNoMoreInteractions(notificationClient);
     }
 
+    @Disabled("HTBHF-2185")
+    @Test
+    void shouldSendNoLongerEligibleEmailWhenEligibleWithNoChildrenOnFeedAndNotPregnant() throws JsonProcessingException, NotificationClientException {
+        // setup some claim variables
+        String cardAccountId = UUID.randomUUID().toString();
+
+        wiremockManager.stubSuccessfulEligibilityResponse(Collections.emptyList());
```

Comment:
No card balance request is made is the eligibility status is determined to be ineligible.